src/core/event/ElementRelativePositionTransformer.js

```diff
@@ -0,0 +1,15 @@
+import autobind from 'autobind-decorator';
+
+export default class ElementRelativePositionTransformer {
```

Comment:
Unless there is more coming, maybe a creating a separate class and file is overkill here? A utility function `transformRelativeElementPosition(element, eventPosition)` would seem more lightweight and appropriate, and such a function could be placed in the same file as a couple of other utilities.